3-platform/order/infra/pom.xml

```diff
@@ -32,5 +32,20 @@
       <artifactId>order-app</artifactId>
       <version>0.0.1-SNAPSHOT</version>
     </dependency>
+
+    <dependency>
+      <groupId>com._6core.lib.java</groupId>
+      <artifactId>reactive-postges-persistent</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.hibernate.reactive</groupId>
+      <artifactId>hibernate-reactive-core</artifactId>
+      <version>2.2.2.Final</version>
+    </dependency>
```

Comment:
I think it should be in reactive-postgres-persistent library

p.s. we need to get rid of it. It uses another reactive library and probably creates conflict. 
It uses `Uni` and `Multi` objects (not Flux, Mono). 
https://hibernate.org/reactive/documentation/2.2/reference/html_single/

p.s.s. we can go with your first version with @Transient. I checked and relationships didn't support by current version of r2dbc. 
```suggestion
```

Reply:
Ok @sokimaaa, without it there are no ORM annotations, only Id, Table, Column, 
jakarta.persistence - entity, one to many, enumerated..